xclim/data/fr.json

```
@@ -733,42 +732,42 @@
   "HURS_FROMDEWPOINT": {
     "long_name": "Humidité relative",
     "description": "Calculée à partir de la température et du point de rosée à l'aide de la pression de vapeur saturante, laquelle fut calculée en suivant la méthode {method}.",
-    "title": "Humidité relative à partir du point de rosée",
-    "abstract": "Calcul de l'humidité relative à partir de la température et du point de rosée à l'aire de la pression de vapeur saturante."
+    "title": "Humidité relative calculée à partir de la température et du point de rosée",
+    "abstract": "Calcul de l'humidité relative calculée à partir de la température et du point de rosée à l'aide de la pression de vapeur saturante."
```

```suggestion
    "abstract": "Humidité relative calculée à partir de la température et du point de rosée à l'aide de la pression de vapeur saturante."
```
Juste pour enlever `Calcul de l'humidité calculée`